src/theme/Footer/FooterLegal/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import styles from './FooterLegal.module.css'
+export default function FooterLegal() {
+  return (
+    <div className={styles.footer__legal}>
+      <div>
+        <img src="/img/icons/ensono-logo.svg" alt="Ensono logo"/> 
+      </div>
+      <div>
+        <span>Privacy Policy © {new Date().getFullYear()} Ensono, LLC. All rights reserved</span>
+      </div>
+      <div>
+        {
+          listOfSocialMedia.map((socialMedia) => {
+            const {url, Component} = socialMedia
+            return (
+              <a href={url} target="_blank">
+                <Component/>
+              </a>
+            )
+          })}
+      </div>
+    </div>
+  );
+}
+
+const listOfSocialMedia = [ 
+  {
+    Component: () => (<FacebookLogo/>),
+    url: 'https://www.facebook.com/EnsonoIT/'
+  },
+  {
+    Component: () => (<TwitterLogo/>),
+    url: 'https://twitter.com/EnsonoIT'
+  },
+  {
+    Component: () => (<LinkedInLogo/>),
+    url: 'https://www.linkedin.com/company/ensono'
+  }
+]
+
+const FacebookLogo = () => (
+  <svg width="24" height="24" viewBox="0 0 24 24" fill-rule="evenodd"><title>Facebook</title><path d="M9 8h-3v4h3v12h5v-12h3.642l.358-4h-4v-1.667c0-.955.192-1.333 1.115-1.333h2.885v-5h-3.808c-3.596 0-5.192 1.583-5.192 4.615v3.385z"></path></svg>
```

Comment:
`fill-rule` should be `fillRule` but I think you can just remove it anyway as I don't think it's needed in any of these `svgs`.

Reply:
remove the fillRule in